Libraries/dotNetRDF.Data.DataTables/dotNetRDF.Data.DataTables.csproj

```diff
@@ -30,6 +30,8 @@
     <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
     <CodeAnalysisRuleSet>..\..\dotnetrdf.ruleset</CodeAnalysisRuleSet>
     <RootNamespace>VDS.RDF.Data.DataTables</RootNamespace>
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
```

Comment:
I added full debug symbols for my first OpenCover attempt. Not sure if it's required for dotCover...